src/frontend/components/QuillEditor/QuillEditor.jsx

```diff
@@ -1,28 +1,25 @@
 import React from "react";
 import ReactQuill from "react-quill";
-import { QuillToolbar, modules, formats } from "./EditorToolbar";
+import { modules, formats } from "./ToolbarSetup";
 import "react-quill/dist/quill.snow.css";
 import "react-quill/dist/quill.core.css";
 import "./QuillEditor.css";
-const QuillEditor = ({ content, noteDataDispatch }) => {
 
+const QuillEditor = ({ content, noteDataDispatch }) => {
   const handleChange = value => {
     noteDataDispatch({ type: "SET_CONTENT", payload: value });
   };
 
   return (
     <>
```

Comment:
Its good that you removed the extra div and made use of fragments. Removed the load from DOM.